task 3.py

```diff
@@ -0,0 +1,7 @@
+x = list(map(int, list(iput())))
+left = sum(x[0:-3])  #с помощью среза получаем от 0 элемента до 3 с конца
+right = sum(x[-3:])  #с помощью среза получаем сумму последних трёх чисел
```

Comment:
С положительными индексами тоже бы сработало. Передавать ноль, если выберете срез сначала, не обязательно.
```suggestion
left = sum(x[:3])  #с помощью среза получаем от 0 элемента до 3 с конца
right = sum(x[3:])  #с помощью среза получаем сумму последних трёх чисел
```